common/src/main/java/com/wynntils/models/containers/containers/ProfessionCraftingContainer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.containers.containers;
+
+import com.wynntils.models.containers.Container;
+import com.wynntils.models.containers.type.ContainerBounds;
+import com.wynntils.models.containers.type.HighlightableProfessionProperty;
+import com.wynntils.models.profession.type.ProfessionType;
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class ProfessionCraftingContainer extends Container implements HighlightableProfessionProperty {
```

Comment:
Crafting containers were one of the types I thought could be added, but didn't serve a purpose at the time so I left them out. I was thinking more of having an abstract CraftingContainer like PersonalStorage and Reward, then each crafting container having an associated ProfessionType. That would contain a list of the material and ingredient slots for anything we would need them for.

If you want to leave it like this for now that's fine as what I suggested doesn't serve a purpose yet, unless you wanted to modify this to say on the crafting screen it would instead become a toggle to only highlight ingredients in your inventory that are relevant to the current crafting station. So if I open the alchemism station, I would toggle the button to highlight alchemism ingredients

Reply:
I won't complicate it further, until we have an idea what to use it for. Made the inventory highlighted though.